<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta charset="UTF-8">
</head>
<body>
<h2>Vol 1950: 5 Dec. 2008</h2>
<h3>Table of contents</h3>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.1</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>Actual usage of biological nomenclature and its implications for data integrators; a national, regional and global perspective</h3>
<i>Charles Hussey</i>, <i>Yde De Jong</i>, <i>David Remsen</i><h4>Pages: 5-8</h4>
<h4>Abstract</h4>
<p>Biological names play an important role in resource identification and as anchors for all sorts of associated information. This is borne out in ever-expanding online resources but the ways in which names are stored and presented give rise to challenges and pitfalls that can lead to missed or misinterpreted information. These resources must serve a variety of users and keep abreast of changes in nomenclature and systematics. Observations on the use of biological names are presented and some solutions to the challenges are offered.</p>
<h4>Keywords</h4>
<p>Biodiversity; Digital data bases; Nomenclators</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.2</p>
<h3>The Linnean foundations of zoological and botanical nomenclature</h3>
<i>Otto Kraus</i><h4>Pages: 9-20</h4>
<h4>Abstract</h4>
<p>In contrast to primary taxonomic research, nomenclature is a tool to be used in information retrieval and with the aim of ensuring world-wide understanding. Radical changes to its guiding principles would be counterproductive as they would produce disorder in a system of internationally accepted conventions that have developed and matured over decades. Classifications are ordering systems. They are even more utilitarian in function and should not be constantly adapted to potentially ephemeral phylogenies. In principle, the Linnaean foundations of nomenclature and classification survive. Attempts to introduce fundamental changes or even alternatives were proposed a century too late to get enforced; they therefore failed.</p>
<h4>Keywords</h4>
<p>Classification; History of systematics; Linnaean hierarchy; Nomenclature</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.3</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>Zoological vs. botanical nomenclature: a forgotten BioCode experiment from the times of the Strickland Code</h3>
<i>Alessandro Minelli</i><h4>Pages: 21-38</h4>
<h4>Abstract</h4>
<p>A code intended to apply to both zoological and botanical nomenclature was drafted in 1842–43 by a commission appointed by the fourth Congress of Italian Scientists on request of the zoologist Carlo Luciano Bonaparte, as a reaction to the recently published Strickland Code (1842). Large excerpts from the latter document and an English translation of documents related to the Italian initiative are presented in appendices.</p>
<h4>Keywords</h4>
<p>Bonaparte (Carlo Luciano); Congresses of Italian scientists; History of nomenclature; Italian zoology</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.4</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>ZooBank: Developing a nomenclatural tool for unifying 250 years of biological information</h3>
<i>Richard L. Pyle</i>, <i>Ellinor Michel</i><h4>Pages: 39-50</h4>
<h4>Abstract</h4>
<p>Nomenclature represents the backbone upon which virtually all biological information is organized. However, the practice of zoological nomenclature has changed relatively little since its start in 1758. As modern technology changes the paradigm under which modern scientists exchange information, there is increasing need to capitalize on these same technologies to fortify nomenclature. ZooBank has been proposed as the official registry of names and nomenclatural acts, in zoology, as well as associated published works and their authors, and type specimens. Having a coordinated registry of zoological names, integrated with the existing Code of Zoological Nomenclature, will allow increased efficiency of communication among biologists, and enhanced stability of names. Such a registry would encompass two distinct realms, each with their own set of challenges. Retrospective registration involves the monumental task of aggregating and validating two and a half centuries of existing names, whereas prospective registration must be tightly integrated with the future paradigm in which scientific names are created and managed under new models of publication. The prototype of ZooBank has been hosted at Bishop Museum during its initial development phase. Following the lead of standard-setting bodies in biodiversity informatics, Life Science Identifiers (LSIDs) have been selected for use as the globally unique identifiers for ZooBank registration entries. The first ZooBank LSIDs were issued on January 1 st , 2008, and included five new fish species described in a work published that same day, as well as all 4,819 names established in the 10 th Edition of Linnaeus’ <i>Systema</i> <i>Naturae.</i> Three alternate scenarios for implementing mandatory registration in ZooBank have been articulated, each incorporating different degrees of coordination between published works and registration events. A robust discussion involving a broad spectrum of practicing zoological taxonomists is required over the next several years to define the specific implementation aspects of ZooBank.</p>
<h4>Keywords</h4>
<p>Linnaeus; Prospective registration; Registration of names and acts; Retrospective registration; <i>Systema</i> <i>Naturae</i></p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.5</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>Phylogenetic hypotheses, taxa and nomina in zoology</h3>
<i>Alain Dubois</i><h4>Pages: 51-86</h4>
<h4>Abstract</h4>
<p>Taxonomic paradigms have changed several times during the history of taxonomy, yet a single nomenclatural system, socalled Linnaean, has remained in force all along. It is theory-free regarding taxonomy as it relies on ostensional allocation of nomina to taxa, rather than on intensional definitions of nomina (e.g., “phylogenetic definitions”). Nomina are not descriptions, definitions or theories but simple labels designating taxa. Both for theoretical and practical reasons, this system should be maintained for the allocation and validity of nomina under a cladistic taxonomic paradigm. Whereas taxa can be cladistically defined by <i>apognoses</i> or <i>cladognoses,</i> nomina should remain attached to taxa through onomatophores, combined in some cases with a Principle of Coordination. Under such a system, the allocation of nomina to taxa is automatic, unambiguous and universal, and nomenclature does not infringe upon taxonomic freedom. However, to avoid misunderstandings and to solve some current problems, the current <i>Code</i> of zoological nomenclature should be improved in several respects. The distinction should be made clear between <i>taxonomic</i> <i>categories,</i> which have biological definitions, and <i>nomenclatural</i> <i>ranks,</i> which do not, as they give only a position in a nomenclatural hierarchy: if used consistently under a cladistic paradigm, they simply allow to express hypotheses about successive branchings and sistertaxa relationships. Taxa referred to a given rank in different groups cannot therefore be considered equivalent by any biological criterion. The nomenclatural rules should cover the whole taxonomic hierarchy, which is currently not the case in zoology. The recent strong increase in the number of higher taxa which results from cladistic analyses may quickly lead to chaos and problems in communication if the nomina of these taxa continue to be based on personal tastes and opinions. There is an urgent need for the zoological <i>Code</i> to cover these nomina with automatic and stringent rules leaving no place to subjective interpretation. Just like for those currently covered by the <i>Code,</i> the status of these nomina should be established in their first publication (nomenclatural <i>founder</i> <i>effect).</i> The <i>Code</i> should be protected against alternative nomenclatural systems by rejecting as unavailable all nomina and nomenclatural acts published without respecting the basic Linnaean system of nomenclatural hierarchy of ranks.</p>
<h4>Keywords</h4>
<p>Allocation of nomina to taxa; Apognoses; Cladistic hypotheses; Cladognoses; <i>Code</i>; Definitions of taxa; Diagnoses; Equivalence between taxa; Hypotheses; Linnaean nomenclatural hierarchy; Monosemy; Nomenclatural founder effect; Nomenclatural parsimony; Nomenclatural ranks; Nomenclature; Nomina; Onomatophores; Polysemy; Principle of Coordination; Redundancy; Taxa; Taxonomic categories; Taxonomy</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.6</p>
<h3>Anatomical nomenclature: homology, standardization and datasets</h3>
<i>Gregory D. Edgecombe</i><h4>Pages: 87-95</h4>
<h4>Abstract</h4>
<p>Strictly homology-based character names have the benefit of a consistent, evolutionary basis but must overcome practical problems in terms of the function that names serve as tools for communication. Character names should be fixed at the level of primary (rather than secondary) homology in order to maintain nomenclatural stability between competing phylogenies and to allow characters to potentially re-optimize with the addition of data. Inconsistent rules determine the priority of names for anatomical structures, in marked contrast to the stability and clarity provided by Codes for taxonomic nomenclature. Standardized anatomical nomenclature is amenable to a web-based, ontology-driven framework. Imagery and associated metadata linked to phylogenetic datasets facilitate character documentation, nomenclatural stability, and repeatability without requiring a formal process of typification.</p>
<h4>Keywords</h4>
<p>Primary homology; Priority; Secondary homology; Standardized nomenclature; Typification</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.7</p>
<h3>Morphological nomenclature, between patterns and processes: segments and segmentation as a paradigmatic case</h3>
<i>Giuseppe Fusco</i><h4>Pages: 96-102</h4>
<h4>Abstract</h4>
<p>The words we use for describing biological systems and their transformations through development and evolution can recurrently perform as ‘conceptual traps’, i.e. as representations that limit the possibilities of improving our understanding of the very processes they are called to describe. The main focus of this contribution is on the paradigmatic case of segmentation. Limits and drawbacks of the concept of ‘segment’ are critically discussed. Its value as a descriptive unit does not entitle it as a sensible unit for other uses, as for instance investigating the evolution of the developmental process of segmentation.</p>
<h4>Keywords</h4>
<p>Arthropods; Development; Evo-devo; Evolution; Parasegments; Re-segmentation; Segmental mismatch</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.8</p>
<h3>Plant structure ontology: How should we label plant structures with doubtful or mixed identities? </h3>
<i>Bruce K. Kirchoff</i>, <i>Evelin Pfeifer</i>, <i>Rolf Rutishauser</i><h4>Pages: 103-122</h4>
<h4>Abstract</h4>
<p>This paper discusses problems with labelling plant structures in the context of attempts to create a unified Plant Structure Ontology. Special attention is given to structures with mixed, or doubtful identities that are difficult or even impossible to label with a single term. In various vascular plants (and some groups of animals) the structural categories for the description of forms are less distinct than is often supposed. Thus, there are morphological misfits that do not fit exactly into one or the other category and to which it is difficult, or even impossible, to apply a categorical name. After presenting three case studies of intermediate organs and organs whose identity is in doubt, we review five approaches to categorizing plant organs, and evaluate the potential of each to serve as a general reference system for gene annotations. The five approaches are (1) standardized vocabularies, (2) labels based on developmental genetics, (3) continuum morphology, (4) process morphology, (5) character cladograms. While all of these approaches have important domains of applicability, we conclude that process morphology is the one most suited to gene annotation.</p>
<h4>Keywords</h4>
<p>Character cladogram; Continuum; Evo-devo; Flower; Flowering plants; Gene ontology; Intermediate organs; Leaf; Morphological nomenclature; Organ identity; Plant ontology; Plant structure ontology; Root</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.9</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>Learning from Linnaeus: towards developing the foundation for a general structure concept for morphology</h3>
<i>Lars Vogt</i><h4>Pages: 123-152</h4>
<h4>Abstract</h4>
<p>Morphology has fundamental problems regarding aperspectival objectivity of its data—morphological terminology is often based on homology assumptions, lacks standardization, and has problems with comparability, reproducibility, and transparency. This is astonishing given that with his sexual system Linnaeus had already established a high degree of aperspectival objectivity in morphology that unfortunately has been lost subsequently. In the first part of the article a brief introduction to the history of classification is given that provides an answer to the question why morphology only initially has been gripped by the general trend towards objectification that started in the seventeenth century. The conceptual shortcomings of Aristotle’s concept of essences and its link to the definition of species and taxa in natural philosophy play an important part in this development. The only solution to the problem of essences was to link it to the evolutionary concept of homology, which explains why morphological terminology today often rests on homology assumptions. By taking a closer look at Linnaeus’ sexual system, basic principles for developing a general structure concept for morphology are discussed, which would provide the conceptual basis for establishing a high degree of aperspectival objectivity for morphological data. The article concludes with discussing the role of data bases and ontologies for developing a data standard in morphology. A brief introduction to the basic principles of Resource Description Framework (RDF) ontologies is given. A morphological ontology has high potential for establishing a general morphological structure concept if it is developed on grounds of the following principles: morphological terms and concepts must be defined taxon-independently, homology-free, preferably purely anatomically, and if functionally only by clearly indicating the trait’s active participation in a specific biological process.</p>
<h4>Keywords</h4>
<p>Aperspectival objectivity; Bio-ontology; Essentialism; Morphological data; Linguistic problem of morphology; RDF; Standardization</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1950.1.10</p>
<h3>Principles for a names-based cyberinfrastructure to serve all of biology</h3>
<i>David J. Patterson</i>, <i>Sarah Faulwetter</i>, <i>Alexey Shipunov</i><h4>Pages: 153-163</h4>
<h4>Abstract</h4>
<p>The rapidly growing amount of biological data on the internet and the increasing need for large-scale analyses mandate improvements to the management of taxon-centric information. This information, traditionally managed by taxonomists, is now transforming into a web-based infrastructure. The complexity and narrative quality of the biological sciences require an information management framework that is sensitive to the scale, richness, character, and heterogeneity of the discipline. Given that the names of organisms offer us a nearly universal system for indexing biological data objects, a names-based cyberinfrastructure has the capacity to index the totality of available biological information and to aggregate taxon-centric data over a broad scale. In order to serve its role, this infrastructure should incorporate thirteen principles that are proposed here.</p>
<h4>Keywords</h4>
<p>Biodiversity informatics; Taxonomic intelligence; Encyclopedia of Life; Data management; Taxonomy</p>
</div>
</body>
</html>
