<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta charset="UTF-8">
</head>
<body>
<h2>Vol 1945: 28 Nov. 2008</h2>
<h3>Table of contents</h3>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1945.1.1</p>
<h3>Molecular and morphological evidence for two new species of terrestrial planarians of the genus<i> Microplana</i> (Platyhelminthes; Turbellaria; Tricladida; Terricola) from Europe</h3>
<i>Hugh D. Jones</i>, <i>Bonnie L. Webster</i>, <i>D. Timothy J. Littlewood</i>, <i>Jillian C. Mcdonald</i><h4>Pages: 1-38</h4>
<h4>Abstract</h4>
<p>Three species of terrestrial planarian were found in a garden in Yorkshire, <i>Microplana</i> <i>scharffi</i> (Graff 1896), <i>M.</i> <i>terrestris</i> (Müller 1774) and a third <i>Microplana</i> species similar in size and shape to <i>M.</i> <i>terrestris</i> but brown (khaki) rather than grey/black. Mitochondrial cytochome oxidase I sequences show that the three are distinct species, but that the sequences of the khaki specimens are identical with a GenBank sequence under the name “Microplana <i>terrestris”.</i> Photomicrographs of the copulatory apparatus of several specimens of each species, including type material, are presented which confirm that the three species are distinct and show the variation within each species. The khaki species is named as <i>Microplana</i> <i>kwiskea</i> <b>n.</b> <b>sp.</b> Specimens from Spain relating to the GenBank “Microplana <i>terrestris”</i> sequence have been examined and are not of that species, nor are they <i>M.</i> <i>kwiskea.</i> They are considered to be a further species, <i>M.</i> <i>groga</i> <b>n.</b> <b>sp.</b></p>
<h4>Keywords</h4>
<p>Cytochrome oxidase I; <i>cox1;</i> COI; <i>Microplana</i> <i>scharffi;</i> <i>Microplana</i> <i>groga;</i> <i>Microplana</i> <i>kwiskea;</i> <i>Microplana</i> <i>terrestris;</i> Rhynchodemidae; Microplaninae</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1945.1.2</p>
<h3>Allozyme differentiation among populations of the Pyrenean newt <i>Calotriton</i> <i>asper</i> (Amphibia: Caudata) does not mirror their morphological diversification</h3>
<i>Albert Montori</i>, <i>Gustavo A. Llorente</i>, <i>Mario García-París</i><h4>Pages: 39-50</h4>
<h4>Abstract</h4>
<p>The Pyrenean newt (Calotriton <i>asper)</i> is a morphologically diversified species endemic to the Pyrenean mountains (Western Europe) that inhabits fast running streams and mountain lakes. Given its high morphological diversity, the species has been subdivided into at least ten different taxa, subsequently treated by most authors as local forms. Herein, we examined the electrophoretic patterns produced by 20 presumptive allozyme loci in specimens of seven populations distributed over the entire geographic range of the species. Sixteen loci were monomorphic across the sampling area and only four loci were polymorphic. No diagnostic alleles of any population or population set were detected. The average number of alleles per locus was found to be extremely low, between 1.1 and 1.2 ± 0.1. Genetic divergence among populations was minimal, with a maximum divergence of Nei 78 = 0.031. No correlation was shown between genetic and geographic distances (Mantel test: r = -0.29, t = -1.1, p = 0.13). Fst values were low, as would be expected for a nonfragmented population. Estimated gene flow among populations was high, with a Nm = 1.01. Cytochrome b mtDNA sequences from the two populations furthest apart only differed in a single position. According to these genetic/morphological discrepancies, we interpret the observed morphological diversification of <i>C.</i> <i>asper</i> as a product of rapid morphological change under local selection pressure, in response to population specific ecological conditions. The implication of our findings for conservation efforts is that we need to preserve the unique evolutionary processes occurring in single populations or small groups of populations, even if the populations involved cannot be taxonomically differentiated.</p>
<h4>Keywords</h4>
<p>Systematics; Taxonomy; Protein electrophoresis; Genetic differentiation; mtDNA; Morphology; Spain</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1945.1.3</p>
<h3>Dissecting the major African snake radiation: a molecular phylogeny of the Lamprophiidae Fitzinger (Serpentes, Caenophidia)</h3>
<i>Nicolas Vidal</i>, <i>William R. Branch</i>, <i>Olivier S. G. Pauwels</i>, <i>S. Blair Hedges</i>, <i>Donald G. Broadley</i>, <i>Michael Wink</i>, <i>Corinne Cruaud</i>, <i>Ulrich Joger</i>, <i>Zoltán Tamás Nagy</i><h4>Pages: 51-66</h4>
<h4>Abstract</h4>
<p>The Elapoidea includes the Elapidae and a large (~60 genera, 280 sp.) and mostly African (including Madagascar) radiation termed Lamprophiidae by Vidal <i>et</i> <i>al.</i> (2007), that includes at least four major groups: the psammophiines, atractaspidines, lamprophiines and pseudoxyrhophiines. In this work, we reviewed the recent taxonomic history of the lamprophiids, and built a data set including two nuclear protein-coding genes (c-mos and RAG2), two mitochondrial rRNA genes (12S and 16S rRNA) and two mitochondrial protein-coding genes (cytochrome <i>b</i> and ND4) for 85 species belonging to 45 genera (thus representing about 75% of the generic diversity and 30% of the specific diversity of the radiation), in order to clarify the phylogenetic relationships of this large and neglected group at the subfamilial and generic levels. To this aim, 480 new sequences were produced. The vast majority of the investigated genera fall into four main monophyletic clusters, that correspond to the four subfamilies mentioned above, although the content of atractaspidines, lamprophiines and pseudoxyrhophiines is revised. We confirm the polyphyly of the genus <i>Stenophis,</i> and the relegation of the genus name <i>Dromophis</i> to the synonymy of the genus name <i>Psammophis.</i> <i>Gonionotophis</i> <i>brussauxi</i> is nested within <i>Mehelya.</i> The genus <i>Lamprophis</i> Fitzinger, 1843 is paraphyletic with respect to <i>Lycodonomorphus</i> Fitzinger, 1843. <i>Lamprophis</i> <i>swazicus</i> is the sister-group to <i>Hormonotus</i> <i>modestus,</i> and may warrant generic recognition. Molecular data do not support the traditional placement of <i>Micrelaps</i> within the Atractaspidinae, but its phylogenetic position, along with that of <i>Oxyrhabdium</i> (previously considered to belong to the Xenodermatidae), requires additional molecular data and they are both treated as Elapoidea <i>incertae</i> <i>sedis.</i> The interrelationships of Psammophiinae, Atractaspidinae, Lamprophiinae, Pseudoxyrhophiinae, <i>Prosymna</i> (13 sp.), <i>Pseudaspis</i> (1 sp.) and <i>Pythonodipsas</i> (1 sp.), <i>Buhoma</i> (2 species), and <i>Psammodynastes</i> (1 sp.) remain unresolved. Finally, the genus <i>Lycognathophis,</i> endemic to the Seychelles, does not belong to the African radiation, but to the Natricidae.</p>
<h4>Keywords</h4>
<p>Alethinophidia; Atractaspidinae; c-mos; cytochrome <i>b</i>; <i>Dromophis</i>; Elapoidea; <i>Gonionotophis</i>; <i>Hormonotus</i>; Lamprophiinae; <i>Lamprophis</i>; <i>Lycodonomorphus</i>; <i>Lycognathophis</i>; <i>Mehelya</i>; <i>Micrelaps</i>; ND4; <i>Oxyrhabdium</i>; Psammophiinae; <i>Psammophis</i>; Pseudoxyrhophiinae; RAG2; <i>Simocephalus</i>; <i>Stenophis</i>; 16S rRNA; 12S rRNA</p>
</div>
<div style="padding:10px;margin:10px;border:1px solid rgb(192,192,192);background-color:#F1FFD7;">
<p>DOI: 10.11646/zootaxa.1945.1.4</p>
<p><img src="../Open-Access-logo-300x145.png" width="100"></p>
<h3>Deletions from the North American harvestman (Opiliones) faunal list: <i>Phalangomma virginicum</i> Roewer, 1949 is a synonym of <i>Erebomaster weyerensis</i> (Packard, 1888) (Travunioidea: Cladonychiidae), and a note on "<i>Crosbycus" goodnightorum </i>Roewer, 1951 (Nemastomatidae)</h3>
<i>William A. Shear</i><h4>Pages: 67-68</h4>
<h4>Abstract</h4>
<p></p>
<h4>Keywords</h4>
<p></p>
</div>
</body>
</html>
